src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java

```diff
@@ -20,6 +22,7 @@
     public boolean isHeaderHidden;
     public StudentAttributes studentToViewPageAs;
     public InstructorAttributes previewInstructor;    
+    private String registerMessage;
```

Comment:
Are these meant to be still `public`?
Also, a couple of these appear to be unnecessary, like `previewInstructor`.